jenkins/jenkinsfile

```diff
@@ -0,0 +1,32 @@
+pipeline {
+    agent {
+         docker {
+            label 'AL2-X64'
+            /* See  
+            https://github.com/opensearch-project/opensearch-build/blob/main/docker/ci/dockerfiles/build.ubuntu18.opensearch.x64.dockerfile
+            for docker image
+            */
+            image 'opensearchstaging/ci-runner:ci-runner-ubuntu1804-build-v1'
+            alwaysPull true
+        }
+    }
+
+    environment {
+        JAVA11_HOME="/opt/java/openjdk-11"
+        JAVA14_HOME="/opt/java/openjdk-14"
+        JAVA17_HOME="/opt/java/openjdk-17"
+        JAVA8_HOME="/opt/java/openjdk-8"
+        JAVA_HOME="/opt/java/openjdk-14"
```

Comment:
For 2.0 this needs to be switched to JDK 17, no?

Reply:
We can merge this if it looks fine and then I can backport https://github.com/opensearch-project/OpenSearch/pull/2656 to 2.0 and 2.x.